protractor-docker/run-tests.sh

```diff
@@ -0,0 +1,12 @@
+docker -v
+docker-compose -v
+
+npm install 
+
+docker-compose up -d --remove-orphans
+docker-compose scale chromenode=5
+
+./node_modules/.bin/protractor docker-selenium-grid-conf.js
```

Comment:
Referenced the above as a shell script file and "npm install" as part of the "pretest" script.
Would still keep it in the .md file so that the reader knows what to do.

@cnishina  What do you think ?